data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1845 @@
+			`	He comes back down and says, "Request support equipment to get through safely, we must." Using a communication device, he calls for support from Heliarch agents, warning them about the sandstorm.`
+			`	Shortly after, several dozen Heliarch agents are there, most of them working to keep the sand out, while a few follow you two inside with some form of cutting tool, which they use to surgically remove what Tugroob was slowly examining before: several extremely thin wires, connected to some type of machinery hidden within the wooden beams.`
+			`	After they're done, the agents storm the place but find nobody. What they do find are dozens upon dozens of crates filled to the brim with Heliarch staves and blaster guns.`
+			`	They escort you two back outside as the agents prepare to bring the smuggled weapons to their own ships. Though the storm is still raging on, you can make out the shape of a few Heliarch Interdictors parked near your own ship right before you take off. You regroup with Iplora and Kirrie, and they ask Tugroob for more details on the ordeal, before they head to their own bunks.`
+				accept
+			label hole2
+			`	You two carefully make your way down the hole. It bottoms out around ten meters below the barn.`
+			`	As you were coming down, Tugroob pointed to the cracked, rocky walls, saying that most of them were unstable, and you should watch out for rock slides.`
+			`	Looking around, you spot several smaller holes on the walls just large enough for a human or Saryd arm. There are also few larger cracks that you two shouldn't have much trouble going through.`
+			choice
+				`	(Reach your arms inside the smaller holes to look for anything there.)`
+				`	(Head for the larger cracks to see if they lead to anything.)`
+					goto rockraider
+			`	You carefully go about putting your arm inside a few of the holes, with Tugroob helping you up for the ones you couldn't reach, but ultimately you find nothing.`
+			`	Many small rocks fell while you were checking the holes, but miraculously you both remain unscathed. Unfortunately, too much sand from the storm above is piling up for you to look around for more holes or the exact location of the larger cracks you saw before.`
+			`	"Dangerous, it will be, to remain here for long, Captain," Tugroob says as he urges you to cling to them. He carefully crawls back up, and you two take refuge inside the barn.`
+			`	Night falls, with no sign of the storm dying down, and after receiving a notification from Iplora, Tugroob informs you that you should both head back to the ship and pick her and Kirrie up. You two meet up with the rest of the group, and the agents head to their bunks.`
+				accept
+			label decline
+			`	"I see," he says, then give you the coordinates they will go to. You drop them off and return to the hangars.`
```

Comment:
Inconsistent pronouns; is it 'he' or 'they'? Assuming 'they';

```suggestion
			`	"I see," they say, then give you the coordinates they will go to. You drop them off and return to the hangars.`
```

Reply:
It's 'he'. Any 'they' instances here are because I missed them when editing this mission.